internal/api/organizationconfig_test.go

```diff
@@ -0,0 +1,226 @@
+package api
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/joshsoftware/peerly-backend/internal/app/organizationConfig/mocks"
+	"github.com/joshsoftware/peerly-backend/internal/pkg/apperrors"
+	"github.com/joshsoftware/peerly-backend/internal/pkg/dto"
+	"github.com/stretchr/testify/mock"
+)
+
+func TestGetOrganizationConfigHandler(t *testing.T) {
+	orgSvc := mocks.NewService(t)
+	getOrganizationConfigHandler := getOrganizationConfigHandler(orgSvc)
+
+	tests := []struct {
+		name               string
+		setup              func(mock *mocks.Service)
+		expectedStatusCode int
+	}{
+		{
+			name: "Success fetching organization",
+			setup: func(mockSvc *mocks.Service) {
+				mockSvc.On("GetOrganizationConfig", mock.Anything).Return(dto.OrganizationConfig{}, nil).Once()
```

Comment:
why empty org response for success? don't we compare the actual result with expected one?